modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -49,3 +49,22 @@ You can force the reconciliation by making any minor change to the DPA itself.
 ----
 $ oc -n openshift-adp get secret/oadp-<bsl_name>-<bsl_provider>-registry-secret -o json | jq -r '.data'
 ----
+
+
+[id="openshift-adp-controller-manager-seg-fault_{context}"]
+== `openshift-adp-controller` segmentation fault
```

Comment:
The [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) prohibit using markup in headings. Also, IBM Style dictates using [sentence case](https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#headings-titles-and-banners) for titles. I recognize that this isn't always possible, but I think that you could easily do one of the following: 
* Use a generic reference for the controller: "OpenShift ADP Controller segmentation fault"
* If the specific object reference is important, rephrase and leave off the backtics (`): "Segmentation fault in/for openshift-adp-controller"